charter.html

```diff
@@ -128,15 +128,22 @@ <h1 id="title">PROPOSED Private Advertising Technology Working Group Charter</h1
               Chairs
             </th>
             <td>
-              <i class="todo">[chair name] (affiliation)</i>
+              <ul>
+                <li>Aram Zucker-Scharff (Washington Post)</li>
+                <li>Sean Turner (sn3rd; Invited Expert)</li>
+              </ul>
             </td>
-          </tr>
+         </tr>
           <tr>
             <th>
               Team Contacts
             </th>
             <td>
-              <span class="todo"><a href="mailto:">[team contact name]</a></span> <i class="todo">(0.1 <abbr title="Full-Time Equivalent">FTE</abbr>)</i>
+              <ul>
+                <li>Sam Weiler</li>
+                <li>Wendy Seltzer</li>
```

Comment:
Wendy is no longer with the W3C, so I don't think it'd be appropriate to list her as a Team contact.

Reply:
Excellent point.  This wasn't true when I opened the PR, but here we are.

```suggestion
```